src/lib/Queue.ts

```diff
@@ -1,5 +1,16 @@
 import { LandRequestStatus, LandRequest, PullRequest } from '../db';
 
+const order = [
```

Comment:
where are you using order?

Reply:
Here: https://github.com/atlassian/landkid/pull/113/files#diff-2104577a08320b88076fc9c0cb711a4a6de3ce58e10bfa067afb271d21db5149R50